.github/workflows/dogfood.yml

```diff
@@ -0,0 +1,52 @@
+name: dogfood merge
+# This workflow is triggered on pushes to master and dogfood branches
+on:
+  push:
+    branches:
+      - master
+      - 'dogfood/*'
+
+env:
+  SLACK_CHANNEL: team-lang-dotnet
+  SLACK_USERNAME: Dogfood build action
+
+jobs:
+  dogfood_merge:
+    runs-on: ubuntu-latest
+    name: Update dogfood branch
+    permissions:
+      id-token: write # required for SonarSource/vault-action-wrapper
+      contents: write # required to grant GITHUB_TOKEN writing permission
+    steps:
+      - name: get secrets
+        id: secrets
+        uses: SonarSource/vault-action-wrapper@v2
+        with:
+          secrets: |
+            development/kv/data/slack webhook | SLACK_WEBHOOK;
+      - name: git octopus step
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        id: dogfood
+        uses: SonarSource/gh-action_dogfood_merge@v1
+        with:
+          dogfood-branch: 'dogfood-on-peach'
+      # Use the output from the `dogfood` step
+      - name: Get the name of the dogfood branch and its HEAD SHA1
+        run: echo "The dogfood branch was `${{ steps.dogfood.outputs.dogfood-branch }}` and its HEAD SHA1 was `${{ steps.dogfood.outputs.sha1 }}`"
+      #slack notifications
+      - name: Notify success on Slack
+        uses: Ilshidur/action-slack@1.6.2
+        env:
+          SLACK_WEBHOOK: ${{ fromJSON(steps.secrets.outputs.vault).SLACK_WEBHOOK }}
+          SLACK_OVERRIDE_MESSAGE: 'Dogfood build for `${{ steps.dogfood.outputs.sha1 }}`: *successful*'
+        with:
+          args: 'Succeed to build dogfood branch'
+      - name: Notify failures on Slack
```

Comment:
Nitpick: `success` on line 38 is singular, but `failures` here is plural.
This seems to reflect the saying that misfortune never comes alone :-)
Anyway, you can ignore this comment and keep it as is, if you want to conform to the yml of the other repos, that have the same message.

Reply:
I'll keep it in sync.